gym/lib/gym/options.rb

```diff
@@ -81,6 +81,10 @@ def self.plain_options
                                      env_name: "GYM_CODE_SIGNING_IDENTITY",
                                      description: "The name of the code signing identity to use. It has to match the name exactly. e.g. 'iPhone Distribution: SunApps GmbH'",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :installer_codesigning_identity,
+                                     env_name: "GYM_INSTALLER_CODE_SIGNING_IDENTITY",
+                                     description: "The name of the code signing identity for Mac Installer to use. It has to match the name exactly. e.g. '3rd Party Mac Developer Installer: SunApps GmbH'",
```

Comment:
We should make it more clear that that's only required if you want an `pkg` file for Mac builds.